build.gradle

```diff
@@ -11,10 +11,11 @@ buildscript {
         }
```

Comment:
delete this? -per https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=android